docs/04-architecture/supported-browsers.mdx

```diff
@@ -57,10 +57,6 @@ Next.js allows you to use the latest JavaScript features out of the box. In addi
 - [Class Fields](https://github.com/tc39/proposal-class-fields) and [Static Properties](https://github.com/tc39/proposal-static-class-features) (part of stage 3 proposal)
 - and more!
 
-### Server-Side Polyfills
```

Comment:
`fetch` is not polyfilled anymore (not needed), but I think this section could be its own page anyway.